.github/workflows/build_pex.yml

```diff
@@ -15,12 +15,14 @@ jobs:
   build_pex:
     name: Build PEX
     runs-on: ubuntu-20.04
```

Comment:
I think we were running 20.04 because of python2.7 so this can be upgraded? or are we're not ready to jump to 22.04 in order to run 3.6?

Reply:
Yeah, it looks like 3.6 is only available on 20.04: https://github.com/actions/python-versions/blob/main/versions-manifest.json#L5373